server/src/test/utils/StringBuilder.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { assertThat, is } from 'hamjest';
+import { StringBuilder } from '../../utils/StringBuilder';
+
+describe('StringBuilder', () => {
+  const INITIAL_TEXT = 'initial_text';
+
+  let stringBuilder: StringBuilder;
+
+  beforeEach(() => {
+    stringBuilder = new StringBuilder();
+    stringBuilder.append(INITIAL_TEXT);
+  });
+
+  it('append should add text at the end', () => {
+    const appendedText = 'appendedText';
+    stringBuilder.append(appendedText);
+    assertThat(stringBuilder.toString(), is(INITIAL_TEXT.concat(appendedText)));
```

Comment:
We can avoid writing `is`. 
`assertThat(stringBuilder.toString(), INITIAL_TEXT + appendedText));`